src/pages/About/About.jsx

```diff
@@ -11,7 +11,7 @@ function About() {
       <motion.div
         animate={{ x: 10, scale: 1 }}
         initial={{ x: -200 }}
-        transition={{ ease: "easeOut", duration: 2 }}
+        transition={{ ease: "easeOut", duration: 1 }}
```

Comment:
using plain numbers is a bad practice. they are called **magic numbers** because nobody knows what they mean

some context: https://en.wikipedia.org/wiki/Magic_number_(programming)

please create a `constants.js` file in the `src` folder and export any const that you need with a proper name. for example, the constants file could look like this:

```js
export const ONE_SECOND=1,
  TWO_SECONDS=2;
```

change all magic numbers in this file